modules/smart-agent_systemd-timers/conf/01-timer-has-not-run-for.yaml

```
@@ -0,0 +1,15 @@
+module: systemd
+name: "timer has not run for"
+transformation: false
```

I think it is safe to give the user the ability to set a transformation if he wants

```suggestion
transformation: true
```